numba/cuda/compiler.py

```diff
@@ -313,890 +282,3 @@ class ExternFunction(object):
     def __init__(self, name, sig):
         self.name = name
         self.sig = sig
-
-
-class ForAll(object):
```

Comment:
Note: `ForAll` was considered a configuration like object so moved to `dispatcher.py`